net/net_demo_installer

```diff
@@ -145,8 +145,13 @@ function sudoExec() {
     sudo -E $@
 }
 
-contiv_network_version="v0.1-05-19-2016.08-34-56.UTC"
+: ${CONTIVNETVERSION?"Please set the Contiv Network version"}
```

Comment:
Let's use the method in setup_k8s_cluster.sh to have a default and let people override as necessary.